zerver/webhooks/github/fixtures/pull_request_review_duplicate_payload.json

```diff
@@ -0,0 +1,515 @@
+{
```

Comment:
Also, you can update the commit message accordingly to have the new file name. Ideally, there is no need to mention which file has been added in the commit message, as this can be seen by simply checking out that commit.